blueman/bluez/AnyBase.py

```diff
@@ -49,6 +49,7 @@ def _on_properties_changed(
 
     def close(self) -> None:
         if self.__signal:
-            self.__bus.signal_unsubscribe(self.__signal)
+            if self.__bus is not None:
+                self.__bus.signal_unsubscribe(self.__signal)
```

Comment:
From the interface there's nothing holding a caller from calling `close` twice. Instead of the condition we could also use an `assert`, causing a crash instead of silently doing nothing if it actually happens. mypy just wants to know that we are aware of the fact that `self.__bus` might be `None`.